src/main/java/frc/robot/commands/ClimberDownCMD.java

```diff
@@ -0,0 +1,37 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.Climber;
+
+public class ClimberDownCMD extends CommandBase {
+    private Climber climber;
+
+    public ClimberDownCMD(Climber climber) {
+        this.climber = climber;
+
+        addRequirements(climber);
+    }
+
+    // Called when the command is initially scheduled.
+    @Override
+    public void initialize() {
+    }
+
+    // Called every time the scheduler runs while the command is scheduled.
+    @Override
+    public void execute() {
+        if (climber.isEnabled())
+            climber.down();
+    }
```

Comment:
Should the command care about enabled here? Or should it always tell the subsystem to go down and the subsystem decides if it will comply?